### ISSUE TITLE
erorr when i press export data its appear could not find a part of the path 

### ISSUE THREAD

**Issue by mohammedrazem** [vawser/Yapped-Rune-Bear#84]
![Yapped - Rune Bear Edition 29_9_2022 4_06_15](https://user-images.githubusercontent.com/114630093/192915514-521e04dd-f0cd-4fe3-8e53-3c6fd5107f95.png)

**Comment by Waaffel** [comment]
Right, so I fixed it by renaming the file in ExampleMod named "Names" and renamed it to "CVS". This fixed my issue.
